server/src/main/java/org/elasticsearch/action/admin/indices/alias/get/GetAliasesResponse.java

```diff
@@ -51,23 +50,16 @@
 
     private ImmutableOpenMap<String, List<AliasMetaData>> aliases = ImmutableOpenMap.of();
     private RestStatus status = RestStatus.OK;
-    private String errorMsg = "";
+    private String errorMessage;
 
-    public GetAliasesResponse(ImmutableOpenMap<String, List<AliasMetaData>> aliases, RestStatus status, String errorMsg) {
+    public GetAliasesResponse(ImmutableOpenMap<String, List<AliasMetaData>> aliases, RestStatus status, String errorMessage) {
         this.aliases = aliases;
-        if (status == null) {
-            this.status = RestStatus.OK;
-        }
-        this.status = status;
-        if (errorMsg == null) {
-            this.errorMsg = "";
-        } else {
-            this.errorMsg = errorMsg;
-        }
+        this.status = status == null ? RestStatus.OK : status;
```

Comment:
when do we get null here? I would prefer to have this logic (default is OK, but also 404 can be provided) in one place, while we have it now in the transport action too. I wonder when this if is needed.

Reply:
did you have a chance to look into this super small comment?